Remove all old technical documentation from gen3.org (#322)
uc-cdis/gen3.org: content/content-deprecated/index.md

---
identifier: content-removed

> title: "Notice: Content Deprecated"

Reviewer: ```suggestion
title: "Notice: Content Updated and Moved"
```

linktitle: content-removed
date: 2026-03-25
g3Teaser:

>   title: Notice - Content Deprecated

Reviewer: ```suggestion
  title: Notice - Content Updated and Moved
```

>   detail: The content you are looking for is no longer available. All Gen3 technical documentation has moved to our new site at docs.gen3.org.  Please review the new documentation there.

Reviewer: I think I would suggest changing this to something a bit more positive, in case people don't read past the first line:

> We have new Gen3 documentation! All Gen3 technical documentation has moved to our new site at [docs.gen3.org](http://docs.gen3.org/). The content you are looking for is likely there. Please review the new documentation, and reach out to us at [support@gen3.org](mailto:support@gen3.org) if you have any trouble.

>   detail: The content you are looking for is no longer available. All Gen3 technical documentation has moved to our new site at docs.gen3.org.  Please review the new documentation there.

Reviewer: ```suggestion
  detail: We have new Gen3 documentation! All Gen3 technical documentation has moved to our new site at docs.gen3.org. The content you are looking for is likely there. Please review the new documentation, and reach out to us at support@gen3.org if you have any trouble.
```
